sentences/de/binary_sensor_HassGetState.yaml

```diff
@@ -384,3 +384,16 @@ intents:
           domain: binary_sensor
           device_class: problem
           state: "on"
+
+      # Safety
+      - sentences:
+          - "(ist|sind) <name> {bs_safety_states:state} [<area>]"
```

Comment:
i think sind is not needed here, since we are responding with `einzeln_janein`.
if we want to ask in plural, we need also to respond with `irgendeins` or `alle`

Reply:
Yes and no. The name itself could be a plural even if it's just one entity or maybe a group of sensors.
In the end it's just a translation of the English version where there's also both. But I see possibilities where 'sind' may be required even if it's unlikely.

Reply:
mhhh ... ok, this somehow convince me. maybe we should have at least one test sentence which uses "sind"

Reply:
I've added a test for a sentence with plural.